### ISSUE TITLE
Blueprints Not Importing from Community Forums

### ISSUE THREAD

**Issue by IIAIronWolf** [home-assistant/frontend#8175]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

I have tried to import blueprints from the community forums but am receiving a "URL invalid" error.
## Expected behavior

I expected the blueprint to simply be added from the forums as it is supposed to. 
## Steps to reproduce

Find blueprint in the HA forums and copy the link from the page. Also try copying the link from the exact comment that has blueprint. 
Receive url invalid error. 
## Environment

- Home Assistant release with the issue: 2021.1.3
- Last working Home Assistant release (if known): NA
- Browser and browser version: iOS App (public release)
- Operating system: Home Assistant OS
- Frontend version: 20201229.1

## State of relevant entities

```yaml

```

## Problem-relevant frontend configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

## Additional information

Link used to test: https://community.home-assistant.io/c/blueprints-exchange/53

**Comment by frenck** [comment]
Please provide the URL causing the issue.

Thanks 👍

**Comment by IIAIronWolf** [comment]
> Please provide the URL causing the issue.
> 
> 
> 
> Thanks 👍 

I have updated the issue with the link.

**Comment by frenck** [comment]
@IIAIronWolf That is not a blueprint linked, that a general link to the full forum category.

You can only import specific blueprints. For example:

`https://community.home-assistant.io/t/zha-ikea-five-button-remote-for-lights/253804`

**Comment by IIAIronWolf** [comment]
After clicking said link, I noticed it does not link directly to the blueprint in question. 
I have noticed this issue on the Wyze Community forums as well. If you hit the share button on an iPhone and copy the link, it does not go to the page you're on, but rather to the topic link. 
After copying the exact link from the browser manually, this is now working. 
This seems to be an issue with the provider of the forums for Home Assistant and Wyze in relation to iPhones. 

Since this is no longer the issue I suspected, I will close this.
